ci/Makefile

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Since we now have `make`-targets for the pipeline creation, can you please adjust the related section in the README to use it, instead of executing the script directly?

https://github.com/cloudfoundry/app-autoscaler-release/blob/49e6ef2e5a24ccd126568fbe48f8bd628f58111d/ci/README.md?plain=1#L26-L35